src/components/CreateForm.jsx

```diff
@@ -60,20 +65,35 @@ const CreateForm = function ({createPost, savePost, post, cancelForm}) {
             <form>
                 <div className='mb-3'>
                     <label className='form-label'>Title</label>
-                    <input maxLength={maxLengthTitle} value={title || ''} onChange={e => setTitle(e.target.value)} type="text" className='form-control'/>
-                    {isShowError &&
+                    <input 
+                        maxLength={maxLengthTitle} 
+                        value={title} 
+                        onChange={e => setTitle(e.target.value)} 
+                        type="text" 
+                        className='form-control'/>
+                    {isValidText &&
                         <span className="text-danger">Maximum field length exceeded</span>
                     }
                 </div>
                 <div className='mb-3'>
                     <label className='form-label'>Content</label>
-                    <textarea  value={content} onChange={e => setContent(e.target.value)} type="text" onKeyPress={handleKeyPress} className='form-control'></textarea>
-                    {isShowContentError &&
+                    <textarea  
+                        value={content} 
+                        onChange={e => setContent(e.target.value)} 
+                        type="text" 
+                        onKeyPress={handleKeyPress} 
+                        className='form-control'>
+                    </textarea>
+                    {isValidContent &&
                         <span className="text-danger">Note text exceeded by {content.length - maxLengthContent} characters</span>
                     }
                 </div>
                 <div className='d-flex'>
-                <button disabled={!title || content.length >= 1000} className='btn btn-primary' onClick={isEdit ? save : addPost}>{isEdit ? 'Save' : 'Create'}</button>
```

Comment:
Validation rules can change. Propose to keep validation methods in `isValidText` and `isValidContent`

Reply:
done